roles/load-balancers/manage-haproxy/tasks/install.yml

```diff
@@ -6,13 +6,7 @@
       package:
         name: '{{ item }}'
         state: installed
-      with_items:
-        - haproxy
-        - openssl-devel
-        - firewalld
-        - python3-firewall
-        - libsemanage-python
-        - policycoreutils-python
+      loop: '{{ lb_install_packages|flatten(levels=-1) }}'
```

Comment:
the loops isn't necessary anymore now that `package` actually takes a single string **or a list** of packages to install - hence you can just feed the `lb_install_packages` to the `name` parameter above

Reply:
Code updated